tripper/utils.py

```diff
@@ -128,10 +128,22 @@ def parse_literal(literal: "Any") -> "Literal":
     if isinstance(literal, Literal):
         return literal
 
+    if hasattr(literal, "lang"):
+        lang = literal.lang
+    elif hasattr(literal, "language"):
+        lang = literal.language
+
+    if not lang and hasattr(literal, "datatype"):
+        datatype = (
+            literal.datatype
+            if literal.__class__ == str
```

Comment:
```suggestion
            if literal.datatype.__class__ == str
```
(I'm not sure about this, please check.)

Reply:
No need for that change.

What I don't understand is the logic: why do you want to assign `literal.datatype` straight off only when `literal` is a `str`, and `str(literal.datatype)` for any other case?

Reply:
You are completely right. There is no need for that conditional, we can just `str(literal.datatype)` for all cases. The reason we need the `str()` function is that `literal` may be an instance of `rdflib.Literal` whos `datatype` attribute is not a string, but a `rdflib.URIRef` object.

Reply:
Thank you!